templates/kubernetes/index.html

```diff
@@ -55,7 +54,7 @@ <h2>Cloud Native Platform</h3>
     </div>
     <div class="col-8">
       <h2>Google&nbsp;GKE</h2>
-      <p>Google and Canonical together enable smooth hybrid operations between Google&rsquo;s Container Engine (GKE) service with Ubuntu worker nodes, and Canonical&rsquo;s Distribution of Kubernetes. Choose Canonical&rsquo;s Kubernetes to be sure your container workloads can migrate to GKE thanks to our kernel-to-k8s&nbsp;alignment.</p>
+      <p>Google and Canonical together enable smooth hybrid operations between Google&rsquo;s Container Engine (GKE) service with Ubuntu worker nodes, and Canonical&rsquo;s Distribution of Kubernetes &reg;. Choose Canonical&rsquo;s Kubernetes to be sure your container workloads can migrate to GKE thanks to our kernel-to-k8s&nbsp;alignment.</p>
```

Comment:
I know this is the most minor thing ever, but I think the space is unnecessary

Reply:
But it's there to stop 'alignment' being orphaned :(

Reply:
@grahambancroft oh sorry I meant just before the registered trademark symbol